roles/haproxy/templates/etc/haproxy/haproxy_openstack.cfg

```diff
@@ -56,17 +56,17 @@
       endpoints.neutron.haproxy.prefer_primary_backend,
       endpoints.neutron.haproxy.health_check,
       endpoints.neutron.haproxy.balance
-    ],
-    [
-      'cinder',
-      endpoints.cinder.port.backend_api,
-      endpoints.cinder.port.haproxy_api,
-      endpoints.cinder.haproxy.prefer_primary_backend,
-      endpoints.cinder.haproxy.health_check,
-      endpoints.cinder.haproxy.balance
     ]
   ]
 -%}
+{% if cinder.enabled|default('False')|bool -%}
```

Comment:
should default to true to match expectations set in cinder role - https://github.com/blueboxgroup/ursula/blob/master/roles/cinder-common/defaults/main.yml#L9